README.md

```diff
@@ -13,6 +13,8 @@ Free SVG icon font for popular brands. See them all on one page at <a href="http
 <a href="https://packagist.org/packages/simple-icons/simple-icons-font"><img src="https://img.shields.io/packagist/v/simple-icons/simple-icons-font?logo=packagist&logoColor=white" alt="Build status" /></a>
 </p>
 
+> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md) before using icons from Simple Icons.
+
 ## Setup
 
```

Comment:
In the meantime, I'd say changing it to this :arrow_down: makes it consistent with the main repository. That way we can merge this as is before the next release for sure, and look into moving it up later.

Agreed @adamrusted?

```suggestion
## Setup

> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md) before using icons from Simple Icons.
```